Without backports.zoneinfo some tests for Python 3.7 are just skipped.  It is
not worth packaging backports.zoneinfo because Python 3.7 will be obsoleted
soon anyway.

--- Babel-2.11.0/tox.ini.orig
+++ Babel-2.11.0/tox.ini
@@ -8,7 +8,6 @@
     pytest>=6.0
     pytest-cov
     freezegun==0.3.12
-    backports.zoneinfo;python_version<"3.9"
     tzdata;sys_platform == 'win32'
 whitelist_externals = make
 commands = gmake clean-cldr test
